ref/general/packaging-deploying.adoc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:seo-title: Packaging and deploying applications
+:seo-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:page-layout: general-reference
+:page-type: general
+= Package and deploy applications with Open Liberty
+
+You can choose how to package and deploy your application with Open Liberty to meet your needs.
+
+== Package, containerize, and deploy using Maven and Docker
+
+One of the easiest ways to get your cloud-native application up and running is to use Maven to package it with the Open Liberty runtime and use Docker to containerize and deploy it.
+The Open Liberty Maven plug-in includes goals that allow you to manage your Open Liberty server and applications.
+Go to the link:https://github.com/OpenLiberty/ci.maven[documentation] for information about how to enable the Open Liberty Maven plug-in for your project.
```

Comment:
maybe move the plug-in info to a new paragraph and also mention dev mode & link to that topic. It's a goal of the maven plugin